tests/c/simple_interp_loop2.c

```diff
@@ -13,7 +13,7 @@
 //     --- Begin jit-pre-opt ---
 //     ..~
 //     guard true, %{{34}}, [%{{0}}, %{{1}}, %{{2}}, %{{3}}, %{{4}}, %{{5}}, %{{34}}]
-//     %{{_}}: ptr = load %{{0}}
+//     %{{_}}: ptr = load %{{1}}
```

Comment:
Isn't this commit already here via https://github.com/ykjit/yk/pull/1368/?

Reply:
Ah, didn't see that PR yet. I guess it is!

Reply:
Will rebase once https://github.com/ykjit/ykllvm/pull/193 goes live.